ci-operator/step-registry/telcov10n/functional/ocp-far-edge-vran-tests/telcov10n-functional-ocp-far-edge-vran-tests-commands.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -e
+set -o pipefail
+
+# Fix user IDs in a container
+~/fix_uid.sh
+
+SSH_KEY_PATH=/var/run/kni-qe-41-ssh-key/ssh-key
+SSH_KEY=~/key
+BASTION_IP_ADDR="$(cat /var/run/bastion-ip-addr/address)"
+JENKINS_USER_NAME="$(cat /var/run/jenkins-credentials/jenkins-username)"
+JENKINS_USER_TOKEN="$(cat /var/run/jenkins-credentials/jenkins-token)"
+
+# Check connectivity
+ping $BASTION_IP_ADDR -c 10 || true
+echo "exit" | curl "telnet://$BASTION_IP_ADDR:22" && echo "SSH port is opened"|| echo "status = $?"
+
+cp $SSH_KEY_PATH $SSH_KEY
+chmod 600 $SSH_KEY
+
+git clone https://github.com/shaior/eco-ci-cd.git --depth=1 -b vran-dep ${SHARED_DIR}/eco-ci-cd
+cd ${SHARED_DIR}/eco-ci-cd/
+ansible-galaxy collection install redhatci.ocp --ignore-certs
+
+# ocp-far-edge-vran-tests
+PARAMS=$(cat <<END_CAT
+LOCK_LABEL_HOST=$lock_label_host&\
+SPOKE_NAME=$spoke_name&\
+TEST_TYPE=$test_type 
+END_CAT
+)
+
+
+PARAMS=$(echo "$PARAMS" | tr -d '\n') 
+ansible-playbook playbooks/launch-jenkins-job.yaml -i inventory \
+ -e username=$JENKINS_USER_NAME \
+ -e token=$JENKINS_USER_TOKEN \
+ -e jjl_job_params="$PARAMS" -e jjl_job_url="https://auto-jenkins-csb-kniqe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-far-edge-vran-tests/" -vvv | tee ${SHARED_DIR}/reporting.txt\
```

Comment:
I think we should add `|| exit 1` to fail if there is an error
```suggestion
 -e jjl_job_params="$PARAMS" -e jjl_job_url="https://auto-jenkins-csb-kniqe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-far-edge-vran-tests/" -vvv | tee ${SHARED_DIR}/reporting.txt || exit 1
```